docs/src/main/paradox/scala/http/routing-dsl/testkit.md

```diff
@@ -3,16 +3,24 @@
 One of Akka HTTP's design goals is good testability of the created services.
 For services built with the Routing DSL Akka HTTP provides a dedicated testkit that makes efficient testing of
 route logic easy and convenient. This "route test DSL" is made available with the *akka-http-testkit* module.
+
+@@@ div { .group-scala }
+
 To use it include the following dependency:
 
-@@@vars
-```sbt
-"com.typesafe.akka" %% "akka-http-testkit" % "$project.version$"
-```
+sbt
+:   @@@vars
+    ```
+    "com.typesafe.akka" %% "akka-http-testkit" % "$project.version$"
+    ```
+    @@@
+
 @@@
```

Comment:
This should be included in both the Java and Scala version and ideally should also show Gradle and Maven snippets:

```
sbt
:   @@@vars
    ```scala
    "com.typesafe.akka" %% "akka-http-testkit" % "$project.version$" $crossString$
    ```
    @@@

Gradle
:   @@@vars
    ```
    compile group: 'com.typesafe.akka', name: 'akka-http-testkit_$scala.binary_version$', version: '$project.version$'
    ```
    @@@

Maven
:   @@@vars
    ```xml
    <dependency>
      <groupId>com.typesafe.akka</groupId>
      <artifactId>akka-http-testkit_$scala.binary_version$</artifactId>
      <version>$project.version$</version>
    </dependency>
    ```
    @@@
```